### ISSUE TITLE
Unable to navigate between colors in Editor Font and Background ColorPickers with keyboard

### ISSUE THREAD

**Issue by nelito987** [telerik/kendo-ui-core#6654]
Currently, it is not possible to navigate between colors in Editor Font and Background ColorPickers with the keyboard.

### Reproduction of the problem
1. Go to https://demos.telerik.com/kendo-ui/editor/all-tools
1. Navigate to the Font or BackColor tools in the toolbar
1. Press Down Arrow

### Current behavior
The Font/BackColor tools in the toolbar are opened, but it is not possible to navigate inside in order to select a color.

### Expected/desired behavior

On pressing down the arrow key the color in the top-left corner of the palette should be focused. Then pressing other arrow keys moves the focus to the desired color. Then:
- Pressing Enter selects the color and applies it to the editor`s content
- Pressing Esc closes the dropdown without selecting a color, focus remains on the ColorPicker button. 

### Environment

* **Kendo UI version:** 2021.3.1109
* **Browser:** [all ]
